NFIQ2/NFIQ2Algorithm/src/nfiq2/nfiq2impl.cpp

```diff
@@ -78,12 +78,12 @@ NFIQ2Algorithm::Impl::~Impl()
 
 double
 NFIQ2Algorithm::Impl::getQualityPrediction(
-    const std::vector<NFIQ::QualityFeatureData> &featureVector) const
+    std::unordered_map<std::string, NFIQ::QualityFeatureData> &featureMap) const
 {
 	const double utility {};
```

Comment:
Just noticed this. Is this unused? It's const + default init, so we can't use it, but it is an argument. What does it do? We may want to make this an issue. Similar for deviation since it's not returned.

Update: I think I'm right. Let's make an issue when we review Wed.

Reply:
Will make this an issue